staff/lucas-wloch/challenges/calculadora-app/README.md

```diff
@@ -0,0 +1,27 @@
+# Calculator App
+## INTRO
+
+
+This is a basic web calculator app with all the normal functions.
+You can do sumas, restas, multiplicaciones y divisiones.
```

Comment:
everything is ok, but you should use just one language. Thanks

Reply:
ok Done